app/models/role/types.rb

```diff
@@ -23,6 +23,8 @@ module Role::Types
                             group_and_below_full: :group_and_below_read,
                             group_full: :group_read}
 
+  AllowedPermissionsForSelfRegistration = []
```

Comment:
Ist das die beste Lösung, damit es aus den Wagons überschrieben werden kann? Muss es allenfalls doch als `class_attribute` definiert werden, aber vielleicht nicht auf`Role` sondern auf `Group`? Dann könnte es sogar pro GroupType angepasst werden.

Reply:
Meiner Einschätzung nach gibt die Instanz/der Wagon vor, was maximal erlaubt ist. Es steht immer noch jeder Gruppe die Selbstregistrierung anbietet frei, weniger berechtigte Rollen zu vergeben. 

Ich finde es gut, dass "nichts" erstmal der default ist. Genauso kann ich mir aber vorstellen, dass man mit `:group_read` in eine Wartehallengruppe kommt und so schonmal andere, gleichgesinnte Leute sehen kann.

Eine Anpassung pro GroupType finde ich aber erstmal zu komplex. Wenn sich diese Anforderung ergibt, kann man das umsetzen. Vorerst würde ich es so belassen.